deploy/example_config/cr-config.yaml

```diff
@@ -4,3 +4,4 @@ metadata:
   name: "kd-global-config"
 spec:
   defaultStorageClassName: "ssd"
+  defaultServiceType: "LoadBalancer"
```

Comment:
Soooo what really should we be providing in our example configs here?

Should we have a cr-config-gke.yaml that specifies standard storage class and LoadBalancer service type? And is it worth having anything else?

Reply:
Sure we can.
Currently we don't apply any of the config CRs as part of our deploy. So we can add one for onprem and one for gke i guess?

Reply:
I will mention the config CRs in the readme docs, maybe that is good enough.

I don't know that we need one for on-prem since we don't know as much about what that k8s configuration will look like -- at least as far as storage is concerned; I guess it almost certainly will use NodePort services but the default is then fine for that.

Reply:
Stuff in the recent change looks good w.r.t. that.